src/dune/install_rules.ml

```diff
@@ -262,6 +293,18 @@ let gen_dune_package sctx pkg =
   let dune_version =
     Dune_lang.Syntax.greatest_supported_version Stanza.syntax
   in
+  let lib_entries = Super_context.lib_entries_of_package sctx pkg.name in
```

Comment:
This call used to be delayed under `gen_dune_package ()` below, we know call it eagerly. Probably harmless, but just pointing it out.

Reply:
Thanks for point that out. That seems ok indeed